templates/part.content.php

```diff
@@ -88,6 +88,78 @@ class="icon-toggle keep-unread"
+                                            type="text"
+                                            placeholder="<?php p($l->t('Username')) ?>"
+                                            title="<?php p($l->t('Username')) ?>"
+                                            name="contactName"
+                                            required
+                                            style="width: 200px">
+                                            <div ng-if="App.loading.isLoading('user')"
+                                                ng-class="{'icon-loading-small': App.loading.isLoading('user') }">
+                                            </div>
+                                    </fieldset>
+                                </form>
+
+                                <div style="margin-left: 2em; font-size: 0.85em;"
+                                     ng-if="!(Share.userList.length > 0) && nameQuery && !App.loading.isLoading('user')">
+                                    <?php p($l->t('No users found')) ?>
+                                </div>
+                                <a
+                                    class="icon-category-installed pr-3"
+                                    ng-repeat="user in Share.userList"
+                                    ng-click="Share.shareItem(item.id, user.value.shareWith)">
+                                    {{ user.value.shareWith }}
+                                    <span class="right" style="margin-top: 1.4em; margin-right: 1em"
+                                        ng-class="{'icon-loading-small': App.loading.isLoading(user.value.shareWith), 'icon-checkmark': !App.loading.isLoading(user.value.shareWith) && Share.usersSharedArticles[item.id].includes(user.value.shareWith)}">
+                                    </span>
+                                </a>
+                                <p class="label-group"> <?php p($l->t('Share on social media')) ?> </p>
+                            <div class="row">
+                                <div class="col-4">
+                                    <a target="_blank"
+                                        class="icon-dropdown icon-facebook pr-5"
+                                        ng-href="https://www.facebook.com/sharer/sharer.php?u={{ ::item.url }}"></a>
+                                </div>
+                                <div class="col-4">
+                                    <a target="_blank"
+                                        class="icon-dropdown icon-twitter pr-5"
+                                        ng-href="https://twitter.com/intent/tweet?url={{ ::item.url }}"></a>
+                                </div>
+                                <div class="col-4">
+                                    <a class="icon-dropdown icon-mail pr-5"
+                                        ng-href="mailto:?subject=I wanted you to see this site&amp;body=Check out this site {{ ::item.url }}"></a>
```

Comment:
I believe we should only enable  for example mail if the app is enabled: https://apps.nextcloud.com/apps/socialsharing_email

Reply:
Hi,
What does this method bring more than just sending an email?
Another question, how to implement this application directly in news? 
Thank you